Cargo.toml

```diff
@@ -6,3 +6,7 @@ members = [
 	"crev-lib",
 	"crevsum"
 ]
+
+
+[patch.'crates-io']
```

Comment:
Still needed?

Reply:
Still got to wait on https://github.com/theduke/crates_io_api/pull/19, or we could temporarily point it at the branch of my fork.